src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
```

Comment:
So the correct point to insert the mockserver in marma.conf.js is on the same level as config? That is working, thanks.
```javascript
module.exports = function(config) {
	config.set({
		basePath: '',
		frameworks: ['qunit','openui5'],
		files: [
			{ pattern: 'bower_components/openui5-sap.ui.core/resources/**/*', served: true, included: false, watched: false },
			{ pattern: 'bower_components/openui5-sap.m/resources/**/*', served: true, included: false, watched: false },
			{ pattern: 'bower_components/openui5-themelib_sap_bluecrystal/resources/**/*', served: true, included: false, watched: false },
			{ pattern: 'src/**/*', served: true, included: false, watched: true },
			{ pattern: 'test/**/*.qunit.js', served: true, included: true, watched: true }
		],
		preprocessors: {
			'src/**/*.js': ['coverage']
		},
		openui5: {
			path: 'bower_components/openui5-sap.ui.core/resources/sap-ui-core.js',
			useMockServer: true
		},
		client: {
			openui5: {
				config: {
					theme: 'sap_bluecrystal',
					libs: 'sap.m',
					resourceroots: {
						'sap.m': '/base/bower_components/openui5-sap.m/resources/sap/m',
						'my.namespace': '/base/src/my/namespace'
					},
					themeroots: {
						'sap_bluecrystal': '/base/bower_components/openui5-themelib_sap_bluecrystal/resources'
					}
				},
				mockserver: {
					config: {
						autoRespond: true
					},
					rootUri: 'http://myserver.de:8000/sap/opu/odata/sap/Z_PACKAGE_SRV/',
					metadataURL: '/base/test/service/metadata.xml',
					mockdataSettings: '/base/test/service'
				}
			}
		},
		reporters: ['progress', 'coverage'],
		coverageReporter: {
			type : 'html',
			dir : 'coverage'
		},
		port: 9876,
		colors: true,
		logLevel: config.LOG_INFO,
		autoWatch: true,
		browsers: ['Chrome'],
		singleRun: false
	});
};
```